### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by kevin-10**
- Massive now places all items at 125 str Req
- Unwieldly now places items at 50 stones
- Renamed ComunityCollectionGump to CommunityCollectionGump

### CODE REVIEW

**Comment by Tasanar** [comment]
Will this update force previously spawned items with the Massive and Unwieldy property to update? If not what would be the best way to code it in Serialization.

**Comment by kevin-10** [comment]
It should auto-reflect on all items.  No deserialization changes needed.

File: Scripts/Services/CommunityCollections/BaseCollectionMobile.cs [hunk 1]
```diff
@@ -367,7 +367,7 @@ public virtual void Reward(PlayerMobile player, CollectionItem reward, int hue)
 			
             reward.OnGiveReward(player, this, hue);	
 			
-            player.SendGump(new ComunityCollectionGump(player, this, this.Location));
```
Comment (Argalep) [inline Scripts/Services/CommunityCollections/BaseCollectionMobile.cs]: missing CommunityCollectionGump

Comment (kevin-10) [inline Scripts/Services/CommunityCollections/BaseCollectionMobile.cs]: Ill add the updated csproj in a few.

**Pull request merged**